xpk.py

```diff
@@ -1368,32 +1579,68 @@ def run_gke_node_pool_create_command(args, system) -> int:
-      command += (f' {args.custom_tpu_nodepool_arguments}')
-    elif system.accelerator_type == AcceleratorType['GPU']:
-      command += (' --placement-type=COMPACT ')
-      command += f' --accelerator type={system.gke_accelerator},count={str(system.chips_per_vm)}'
-    task = f'NodepoolCreate-{node_pool_name}'
-    commands.append(command)
-    task_names.append(task)
+    node_pool_name = f'{args.cluster}-np-0'
+    if node_pool_name not in existing_node_pool_names:
+      command = (
+          'gcloud beta container node-pools create'
+          f' {node_pool_name} --region={zone_to_region(args.zone)}'
+          f' --node-locations={args.zone}'
+          f' --cluster={args.cluster} --project={args.project}'
+          f' --machine-type={system.gce_machine_type}'
+          f' --num-nodes={args.num_slices}'
+          f' --accelerator="type={system.gke_accelerator},count={str(system.chips_per_vm)}"'
+          f' --additional-node-network network={args.cluster}-net-1,subnetwork={args.cluster}-sub-1'
+          f' --additional-node-network network={args.cluster}-net-2,subnetwork={args.cluster}-sub-2'
+          f' --additional-node-network network={args.cluster}-net-3,subnetwork={args.cluster}-sub-3'
+          f' --additional-node-network network={args.cluster}-net-4,subnetwork={args.cluster}-sub-4'
+          ' --no-enable-autoupgrade --enable-gvnic --scopes="https://www.googleapis.com/auth/cloud-platform"'
+          f' {capacity_args}'
+      )
+      task = f'NodepoolCreate-{node_pool_name}'
+      commands.append(command)
+      task_names.append(task)
+  else:
+    xpk_print(
+        f'Creating {args.num_slices} node pool or pools of {device_type}\n'
+        f'Underlyingly, we assume that means: {system}'
+    )
+    desired_node_pool_names = [
+        f'{args.cluster}-np-{slice_num}' for slice_num in range(args.num_slices)
+    ]
+
+    for node_pool_name in desired_node_pool_names:
+      if node_pool_name in existing_node_pool_names:
+        continue
+      command = (
+          'gcloud beta container node-pools create'
+          f' {node_pool_name} --node-version={args.gke_version}'
+          f' --cluster={args.cluster}'
+          f' --project={args.project} --node-locations={args.zone}'
+          f' --region={zone_to_region(args.zone)}'
+          f' --num-nodes={system.vms_per_slice}'
+          f' --machine-type={system.gce_machine_type}'
+          f' --host-maintenance-interval={args.host_maintenance_interval}'
+          f' {capacity_args}'
+          ' --scopes=storage-full,gke-default'
+          ' --enable-gvnic --max-pods-per-node 15'
+      )
+      if system.accelerator_type == AcceleratorType['TPU']:
+        command += (' --placement-type=COMPACT ')
+        command += (f' --tpu-topology={system.topology}')
+        command += (f' {args.custom_tpu_nodepool_arguments}')
+      elif system.accelerator_type == AcceleratorType['GPU']:
+        command += (' --placement-type=COMPACT ')
```

Comment:
I see you omitted `--placement-type=COMPACT` for H100s? Is it not needed for GPUs?